test/parallel/test-stream-readable-read-max-hwm.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const common = require('../common');
+const { Readable } = require('stream');
+
+// Make sure that readable completes
+// even when reading buffers larger than
+// MAX_HWM.
+
+// Pick a size larger than MAX_HWM.
+const bufferSize = 10 * 1024 * 1024;
```

Comment:
Can `MAX_HWM` be moved to `lib/internal/streams/state.js` so that it can be imported into this test, to future-proof changes to that value?

Reply:
Fixed